onnxruntime/core/providers/js/operators/gather_nd.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/js/js_kernel.h"
+#include "core/providers/js/js_data_types.h"
+#include "gather_nd.h"
+
+namespace onnxruntime {
+namespace js {
+
+ONNX_OPERATOR_KERNEL_EX(
+    GatherND,
+    kOnnxDomain,
+    13,
+    kJsExecutionProvider,
+    (*KernelDefBuilder::Create())
+        .TypeConstraint("T", JsepSupportedDataTypes())
+        .TypeConstraint("indices", BuildKernelDefConstraintsFromTypeList<TypeList<int64_t>>()),
```

Comment:
there is no such type constraint `indices`. https://github.com/onnx/onnx/blob/main/docs/Operators.md#GatherND

this line can be removed.